src/ansys/fluent/core/workflow.py

```diff
@@ -1558,7 +1557,10 @@ def refresh_after_sleep(_):
                     logger.debug("Already _refreshing, ...")
                 self._refreshing = True
                 logger.debug("Call _refresh_task_accessors")
-                _refresh_task_accessors(self)
+                try:
+                    _refresh_task_accessors(self)
+                except Exception:
```

Comment:
Is there a more specific `Exception` derived class for which we know it is correct to `pass`?

Reply:
Added a comment.